### PR TITLE
Add confirmation dialog when clearing the form

### LINKED ISSUE #188: EMS: no confirmation dialog shown when clearing ringdown form
There's supposed to be a confirmation dialog, but it's not shown when *Clear Form* is clicked.  It just immediately clears.

image

Also, the page should scroll back to the top after clearing to confirm that something happened.  Right now there's no feedback at all.

### PR DESCRIPTION
**Pull request by fwextensions**
- Fix #188.
- Scroll to the top of the form after clearing it to show that the ringdown has been reset.
- Reset margin-top on the alert buttons so they render correctly when nested inside a form.

This is rebased on #193 so that the buttons are laid out correctly.

@cleneman, note that the buttons are stacked, which seems to be the default for this alert component.  

### After

image

### CODE REVIEW

**Review by francisli — APPROVED** [review]

**Comment by francisli** [comment]
Are we waiting on an approval from Callum...?

I noticed #189 as well while testing this...

**Comment by fwextensions** [comment]
I guess waiting on #193, since this depends on it.

I'm inclined to keep a consistent vertical button stack in the dialogs, rather than have one-off horizontal rows when the labels fit.

#189 can be tackled separately, since it's not just CSS tweaks.

**Comment by cleneman** [comment]
Vertical stack looks good to me! More flexible for windows of various widths - thanks, John!

**Pull request merged**
